topics/go/algorithms/strings/reverse/reverse_test.go

```diff
@@ -0,0 +1,27 @@
+package strings_test
+
+import (
+	"github.com/ardanlabs/gotraining/topics/go/algorithms/strings/reverse"
+	"testing"
+)
+
+const succeed = "\u2713"
+const failed = "\u2717"
+
+func TestReverseString(t *testing.T) {
+
+	// Create string for testing.
```

Comment:
Remove this comment

Let's use a table test here
```
tt := []struct {
  name string
  input string
  output string
}{
{
  {"basic", "Hello World", "dlroW olleH"},
}

Then loop through this with the output as expected and the result as got.

Reply:
Completed